plugin/trino-phoenix5/pom.xml

```diff
@@ -15,7 +15,8 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.hbase.version>2.2.7</dep.hbase.version>
+        <dep.hbase.version>2.5.8-hadoop3</dep.hbase.version>
```

Comment:
Yes, this should work.
This slipped my mind earlier. Without logs, I could not come to realize why master was not getting started.